src/service/resources-impl.js

```diff
@@ -1048,9 +1305,8 @@ export class ResourcesImpl {
         }
 
         if (needsMeasure) {
-          const wasDisplayed = r.isDisplayed();
-          r.measure();
-          if (wasDisplayed && !r.isDisplayed()) {
+          const noLongerDisplayed = this.measureResource_(r);
```

Comment:
IMHO, a more logical return from `measureResources_` would be reverse. E.g. "is this a displayed element after measurement"